test/test_data_types.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+
+from stone.ir import (
+    resolve_aliases,
+    Alias,
+    String
+)
+
+class TestDataTypes(unittest.TestCase):
+    """Tests for the data_types module"""
+
+    def test_resolve_aliases(self):
```

Comment:
Would it make sense to move this test to `test_backend` as well? `resolve_aliases` should only be called by the backend.

Reply:
That sounds fine - I just created this module since the function belongs to the `data_types` module and I anticipate there will be more tests added there in the future. Whichever you think is best!

Reply:
Yeah, it's generally reasonable to have test after each file. However it's not how existing tests are organized. And I feel it's kind of confusing to have mixed organization.